### ISSUE TITLE
Contributing additional helper methods for the evernote SDK

### ISSUE THREAD

**Issue by captn3m0** [evernote/evernote-sdk-ruby#15]
As part of an app I'm working on, I extended the `Evernote::EDAM::NoteStore::NoteStore::Client` class to add a few helper methods. The method signatures are as follows:
- `find_notebook(name)` - Find a notebook with given name
- `create_notebook(name)` - Create a notebook with given name, which calls `Evernote::EDAM::Type::Notebook.new()` for you
- `find_or_create_notebook(name)` - Search for a notebook by name, and return either that or create a new notebook and return that. 

Currently the code is abstracted in an EvernoteUtil class of mine, but I'd be willing to try to patch it back to the sdk. However, I'd like to know if this repo even accepts contributions and if such contributions (adding helper methods not in official API) might be accepted.

**Comment by nozpheratu** [comment]
I appreciate all the work that went into this project, but the public API is a little rough around the edges. @sethhitch you guys accepting public contributions at all?

**Comment by captn3m0** [comment]
@nozpheratu I'd say you should take a look at other evernote [API wrappers instead](https://rubygems.org/search?utf8=%E2%9C%93&query=evernote).

**Comment by nozpheratu** [comment]
@captn3m0 I saw the evernote_util project, but I think there should be one go-to gem that everyone uses. Having wrappers on-top of wrappers just needlessly complicates things IMO.

**Comment by captn3m0** [comment]
Yes. I agree. I wish I was conversant with Thrift. The issue is that since its a thrift-based API, its cumbersome to right a gem that does not use Thrift (which screws everything up).

I even wrote a [rant on the Evernote API](https://www.quora.com/What-are-some-poorly-written-APIs/answer/Abhay-Rana).

**Comment by nozpheratu** [comment]
@captn3m0 FWIW, I don't think anybody is conversant with thrift. :/
